packages/language-server/src/__test__/linting.test.ts

```diff
@@ -92,4 +93,21 @@ suite('Linting', () => {
       fixturePathModelIgnore,
     )
   })
+
+  test('Unknown previewFeature', () => {
+    assertLinting(
+      [
+        {
+          message:
+            'The preview feature "huh" is not known. Expected one of: deno, driverAdapters, fullTextIndex, fullTextSearch, metrics, multiSchema, nativeDistinct, postgresqlExtensions, tracing, views, relationJoins.\nIf this is unexpected, it might be due to your Prisma VSCode Extension being out of date.',
```

Comment:
```suggestion
            'The preview feature "huh" is not known. Expected one of: deno, driverAdapters, fullTextIndex, fullTextSearch, metrics, multiSchema, nativeDistinct, postgresqlExtensions, tracing, views, relationJoins.\nIf this is unexpected, it might be due to your Prisma VS Code Extension being out of date.',
```